pretyping/reductionops.ml

```diff
@@ -1462,6 +1458,8 @@ let infer_conv_gen conv_fun ?(catch_incon=true) ?(pb=Reduction.CUMUL)
     in
       if b then sigma, true
       else
+        let x = EConstr.Unsafe.to_constr x in
```

Comment:
Are those expected to stay?

Reply:
As long as the kernel reduction functions take a Constr.t instead of an EConstr.t, I think so.